README.md

```diff
@@ -33,16 +33,16 @@ downloads illustrates the package types for OMI:
 
 Platform | Release | Architecture | SSL   | Filename
 -------- |-------- |------------  | ---   | --------
-Linux    | Debian  | x64          | 1.1.0 | [omi-1.4.1-0.ssl_110.ulinux.x64.deb](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_110.ulinux.x64.deb)
-Linux    | Debian  | x64          | 1.0.0 | [omi-1.4.1-0.ssl_100.ulinux.x64.deb](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_100.ulinux.x64.deb)
-Linux    | Debian  | x64          | 0.9.8 | [omi-1.4.1-0.ssl_098.ulinux.x64.deb](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_098.ulinux.x64.deb)
-Linux    | RPM     | x64          | 1.1.0 | [omi-1.4.1-0.ssl_110.ulinux.x64.rpm](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_110.ulinux.x64.rpm)
-Linux    | RPM     | x64          | 1.0.0 | [omi-1.4.1-0.ssl_100.ulinux.x64.rpm](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_100.ulinux.x64.rpm)
-Linux    | RPM     | x64          | 0.9.8 | [omi-1.4.1-0.ssl_098.ulinux.x64.rpm](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_098.ulinux.x64.rpm)
-Linux    | Debian  | x86          | 1.0.0 | [omi-1.4.1-0.ssl_100.ulinux.x86.deb](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_100.ulinux.x86.deb)
-Linux    | Debian  | x86          | 0.9.8 | [omi-1.4.1-0.ssl_098.ulinux.x86.deb](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_098.ulinux.x86.deb)
-Linux    | RPM     | x86          | 1.0.0 | [omi-1.4.1-0.ssl_100.ulinux.x86.rpm](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_100.ulinux.x86.rpm)
-Linux    | RPM     | x86          | 0.9.8 | [omi-1.4.1-0.ssl_098.ulinux.x86.rpm](https://github.com/Microsoft/omi/releases/download/v1.4.1-0/omi-1.4.1-0.ssl_098.ulinux.x86.rpm)
+Linux    | Debian  | x64          | 1.1.0 | [omi-`*.*.*-*`.ssl_110.ulinux.x64.deb](https://github.com/Microsoft/omi/releases/latest)
```

Comment:
I thought I did this "right" when I first created this, but clearly my intent was not communicated.

See lines 30-32 before this. In particular:

You can download and install OMI from the [Releases] page. **While
version numbers change from release to release, the following
downloads illustrates the package types for OMI:**


It was never the intent that this file be modified from release to release. This was supposed to provide a "snapshot in time" idea of what to find. Probably all the links should just be references to the latest releases page without specific filenames.